packages/framework/azure-client/package.json

```diff
@@ -77,7 +77,7 @@
     "typescript": "~4.5.5"
   },
   "peerDependencies": {
-    "fluid-framework": "^0.58.0"
+    "fluid-framework": "^0.59.2000"
```

Comment:
I didn't realize that next doesn't have the changes that moved this package? So since the package is still in the main client monorepo (release group), this should be 60.1000.